README.md

```diff
@@ -20,12 +20,13 @@ The forth part is applying media queries to the site to make the mobile version
 
 - https://basem909.github.io/microverse-portofolio-project/
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the website is deployed successfully using GitHub pages.
Currently, this is the error message we have on the page:
![Screenshot from 2022-03-28 22-06-49](https://user-images.githubusercontent.com/58692590/160487691-fd8d90ac-db57-4bd6-ba3c-8af13874f8ae.png)

Reply:
Hi Whoistolu,
It is already deployed successfully and working here you could give it a look in the actions section it's all right

Reply:
- [ ] Kindly update your website live demo link in your README.md file
- [ ] I noticed you made a little mistake on the link you used in your README.md file.
- You used this `https://basem909.github.io/microverse-portofolio-project/` and in the photo, you used to reply to the previous reviewer, there is a dash after project in your URL. 
- https://basem909.github.io/microverse-portofolio-project-/
- [ ] Please update your README with this last link with the dash after `project `👍 
- Please see the screenshot below.
![image](https://user-images.githubusercontent.com/50941074/160497038-8d526c18-4648-4720-9cb6-82dff039969d.png)

- [ ] I recommend you always use the correct link wherever you need to, test if all is working before pushing for a review.